ReactiveCocoa/NSObject+KeyValueObserving.swift

```diff
@@ -55,6 +55,34 @@ extension Reactive where Base: NSObject {
 	}
 }
 
+extension Property where Value: OptionalProtocol {
+	/// Create a property that observes the given key path of the given object. The
+	/// generic type `Value` can be any Swift type that is Objective-C bridgeable.
+	///
+	/// - parameters:
+	///   - object: An object to be observed.
+	///   - keyPath: The key path to observe.
+	public convenience init(object: NSObject, keyPath: String) {
+		// `Property(_:)` caches the latest value of the `DynamicProperty`, so it is
+		// saved to be used even after `object` deinitializes.
+		self.init(UnsafeOptionalKVOProperty(object: object, keyPath: keyPath))
+	}
+	
+	// `Property(unsafeProducer:)` is private to ReactiveSwift. So the fact that
+	// `Property(_:)` uses only the producer is explioted here to achieve the same effect.
+	private final class UnsafeOptionalKVOProperty: PropertyProtocol {
```

Comment:
`UnsafeKVOProperty` can be shared, since the only thing different is the transform.

Reply:
Alright, I've changed that